src/nrnoc/multicore.h

```diff
@@ -89,12 +93,12 @@ struct NrnThread {
     int* _v_parent_index;
     Node** _v_node;
     Node** _v_parent;
-    double* _sp13_rhs;           /* rhs matrix for sparse13 solver. updates to and from this vector
-                                    need to be transfered to actual_rhs */
-    char* _sp13mat;              /* handle to general sparse matrix */
-    Memb_list* _ecell_memb_list; /* normally nullptr */
-    Node** _ecell_children;      /* nodes with no extcell but parent has it */
-    void* _vcv;                  /* replaces old cvode_instance and nrn_cvode_ */
+    double* _sparse_rhs; /* rhs matrix for sparse13 solver. updates to and from this vector
+                          need to be transfered to actual_rhs */
+    Eigen::SparseMatrix<double, 1, int>* _sparseMat{}; /* handle to general sparse matrix */
```

Comment:
We give number otherwise we have to declare the enum behind the option and I have no idea to do that without conflict at resolution...